.github/workflows/sdk-release.yml

```diff
@@ -91,3 +91,25 @@ jobs:
           pnpm --filter "@nexeraprotocol/nexera-id-sig-gating-contracts" publish --access=public --no-git-checks --registry https://registry.npmjs.org/
 
           pnpm --filter "@nexeraprotocol/nexera-id-sig-gating-contracts-sdk" publish --access=public --no-git-checks --registry https://registry.npmjs.org/
+
+      - name: Build Tezos
+        env:
+          NPM_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          cd tezos-sig-gating-contracts
+          make install
+          make compile
+          cp compiled/extended_gated_nftminter.json src/compiledcontracts/
+          cd ..
```

Comment:
Could this somehow be moved into a one-liner pnpm command in either the tezos folder `package.json` root of the `package.json`? To call it for example 

```
pnpm --filter @nexeraprotocol/nexera-id-tezos-sig-gating-contracts-sdk prepare
pnpm --filter @nexeraprotocol/nexera-id-tezos-sig-gating-contracts-sdk build
```

Reply:
I agree  - I'll call it `inject_compiled_contracts` because that what it does it compiles the contracts and copy it in the `src` folder (package content).